### ISSUE TITLE
[Accessibility] The values of IsKeyboardFocusable and HasKeyboardFocus fields are incorrect for DomainUpDown/NumericUpdown control in Inspect UIA structure

### ISSUE THREAD

**Issue by John-Qiao** [dotnet/winforms#4123]
* .NET Core Version: 
.NET SDK 6.0.100-alpha.1.20514.15

* Have you experienced this same bug with .NET Framework?: 
No

**More Info:**

1. This issue does not occur in .NET Core 3.1 and .NET Framework.
2. This issue occurs in NumericUpDown control too.

**Problem description:**

The IsKeyboardFocusable field is **false** and the HasKeyboardFocus field is **true** in UIA structure.
![Actual](https://user-images.githubusercontent.com/45864985/96246185-deca0200-0fda-11eb-9b09-b7fabd6cb970.png)

**Expected behavior:**

The IsKeyboardFocusable field is **true** and the HasKeyboardFocus field is **false** in UIA structure. Like it works in .NET Core 3.1 or .NET Framework.
![Expected](https://user-images.githubusercontent.com/45864985/96246194-e25d8900-0fda-11eb-8e61-a7d141ba74f7.png)

**Minimal repro:**

1. Open the attached [TestApp.zip](https://github.com/dotnet/winforms/files/5390801/TestApp.zip) project in VS, run it and launch the Inspect tool.
2. Click on the DomainUpDown edit control and move the mouse pointer to focus on the whole DomainUpDown control.
3. Observe the IsKeyboardFocusable and HasKeyboardFocus fields' value in Inspect UIA structure.
![ReproSteps](https://user-images.githubusercontent.com/45864985/96246174-da9de480-0fda-11eb-8862-d374dc974239.gif)

**Comment by Tanya-Solyanik** [comment]
@John-Qiao - good bug! Do you happen to know when this regressed in 5.0?

**Comment by John-Qiao** [comment]
@Tanya-Solyanik in my test result, this issue still can be reproduced with .NET 5.0.100-preview.2.20175.2 SDK, please check below screenshot:
![NET5 0-P2](https://user-images.githubusercontent.com/45864985/96430275-8aba5a00-1234-11eb-9709-637d47514e5b.png)


And I tried to reproduce this issue with .NET 5.0.100-preview.1.20155.7 SDK too, but the project can only be built in 'dotnet build' command line, it cannot be started after executed the 'dotnet run' command, just nothing happened in below screenshots:

1. Run project when .NET 5.0.100-preview.1.20155.7 SDK installed:
![NET5 0-P1](https://user-images.githubusercontent.com/45864985/96431503-2dbfa380-1236-11eb-91e7-a2af084c54c6.png)

2. Run project when .NET 5.0.100-preview.2.20175.2 SDK installed:
![NET5 0-P2-run](https://user-images.githubusercontent.com/45864985/96431505-30ba9400-1236-11eb-98cf-5a0b7e5c8902.png)

**Comment by merriemcgaw** [comment]
@vladimir-krestov - getting this in the accessibility project and to the in progress link.

**Comment by John-Qiao** [comment]
Verified the issue in .NET 6.0.100-alpha.1.20557.2 SDK from Master branch, this issue has been fixed that the IsKeyboardFocusable field is true in DomainUpDown and NumericUpdown controls now.
![TR2](https://user-images.githubusercontent.com/45864985/98496897-a20dc580-227d-11eb-9183-2086c88b8005.png)
![TR1](https://user-images.githubusercontent.com/45864985/98496898-a3d78900-227d-11eb-86a1-0d304591fa69.png)
